File: esercizi_python/compito_python_4/Corso.py
class Corso:
    def __init__(self, cod, nomecorso, nomedoc):
        self._cod = cod
        self._nomecorso = nomecorso
        self._nomedoc = nomedoc


    def toString(self):
        return str(self._cod) + "\t"+ self._nomecorso +"\t"+ self._nomedoc + "\t"

File: esercizi_java/NetBeansProjects/Universita/src/universita/Corso.java
package universita;

/**
 *
 * @author tobiasacchetto
 */
public class Corso {
    protected int cod;
    protected String nomecorso,nomedoc;
    
    
    public Corso(int cod,String nomecorso,String nomedoc){
        this.cod=cod;
        this.nomecorso=nomecorso;
        this.nomedoc=nomedoc;
    }
    
    public String toString(){
        return nomecorso+"\t"+ cod + "\t"+nomedoc+"\t";
    }
}
